docs/pkg-application-tutorial.md

```diff
@@ -0,0 +1,301 @@
+# Application Packaging Tutorial
+
+[Arbor](https://arbor-sim.org/) is scientific software for neuroscience simulation, with features including:
+
+* A C++ library with a Python interface
+* Distributed exexcution through MPI
+* Multicore simulation
+* Support for both NVIDIA and AMD GPUs
+
+Users of Arbor fall into two camps: those who want to use a version installed on the system, and those who need to build their own copy.
+
+This tutorial walks through configuring and maintaining a uenv recipe and deployment for a Arbor that supports the different users, which should cover most of the aspects of deploying your own uenv.
+
+## Requirements
+
+Before starting, we gather requirements for the use cases of the uenv on the system, in order to understand:
+
+* which packages the uenv will provide;
+* which interfaces the uenv will provide to those packages.
+
+### Supported workflows
+
+For Arbor we wish to support two workflows:
+
+* *Application*: provide optimised builds of Arbor for users to use directly
+* *BYO*: Arbor is under active development, and some users require the ability to build the latest bleeding edge version themselves.
+* *Developer*: Arbor developers use Alps for development. For this we want to provide
+
+Looking at the above, the *BYO* and *Developer* requirements are the same: provide the dependencies required to build Arbor.
+
+### Supported systems
+
+Arbor is well-optimised for both CPU and GPU executation and users of systems with and without accelerators, so we will provide it for the following platforms:
+
+* multicore: `zen2`/`zen3`
+* `gh200`
```

Comment:
Is there a list of the allowed values somewhere?

Reply:
I have added a list, and a link to confluence where there is more detailed documentation.